cmd/internal/planetscale_edge_database.go

```diff
@@ -400,8 +457,33 @@ func (p PlanetScaleEdgeDatabase) printQueryResult(qr *sqltypes.Result, tableName
 	}
 }
 
-// positionAtLeast returns true if position `a` is equal to or after position `b`
-func positionAtLeast(a string, b string) bool {
+func buildVStreamRequest(tabletType psdbconnect.TabletType, table string, shard string, keyspace string, gtid string) *vtgate.VStreamRequest {
```

Comment:
OK, so I see that this *is* a single table and a single shard. So perhaps this answers my question above...

Reply:
Yes! Exactly 😄